templates/partials/question.html

```diff
@@ -16,10 +15,20 @@
   {%- endif -%}
 {%- endset -%}
 
+{% set question_warning %}
+  {%- if question.warning -%}
+    {% call onsPanel({
+      "id": "question-warning-" + question.id,
```

Comment:
The recommended way to concatenate string in Jinja is using a tilde. I know we use `+` in a lot of places but for new code, we should keep it as recommended.
```suggestion
      "id": "question-warning-" ~ question.id,
```

Reply:
Ok. I have updated the other occurrence of this in this file as well.